pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -542,6 +542,47 @@ func Test_InFlightPods(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:                         "popped pod must have empty UnschedulablePlugins",
+			isSchedulingQueueHintEnabled: true,
```

Comment:
let's add another one without scheduling hints.

We should backport that one to all supported versions

Reply:
I guess it would have to be a separate test function, as this is specific to the inflight implementation.

Reply:
I've moved the new test case into `TestPop` and run it with and without hints.